pkg/snapshot/backup.go

```diff
@@ -72,13 +71,16 @@ func CreateBackup(a *app.App) error {
 	includedNamespaces := []string{appNamespace}
 	includedNamespaces = append(includedNamespaces, kotsKinds.KotsApplication.Spec.AdditionalNamespaces...)
 
-	veleroBackup.Name = makeBackupName(a.Slug)
+	veleroBackup.Name = ""
+	veleroBackup.GenerateName = a.Slug + "-"
+
 	veleroBackup.Namespace = kotsadmVeleroBackendStorageLocation.Namespace
 	veleroBackup.Annotations = map[string]string{
 		"kots.io/snapshot-trigger": "manual",
 		// "kots.io/app-slug":         "", // @areed i don't understand why we need both the slug and id here
-		"kots.io/app-id":       a.ID,
-		"kots.io/app-sequence": strconv.FormatInt(a.CurrentSequence, 10),
+		"kots.io/app-id":         a.ID,
+		"kots.io/app-sequence":   strconv.FormatInt(a.CurrentSequence, 10),
+		"kots.io/snapshot-start": time.Now().UTC().Format(time.RFC3339),
```

Comment:
I think this is more like "snapshot requested"

Reply:
:+1: